apps/mull-api/src/app/entities/post.entity.ts

```diff
@@ -30,16 +30,17 @@ export class Post {
   parentPost?: Post;
 
   @Field()
-  @Column()
+  @Column({ nullable: true })
   message: string;
 
   @Field()
   @Column()
   createdTime: Date;
 
-  @Field(/* istanbul ignore next */ () => [Media], { nullable: true })
-  @OneToMany(() => Media, (media) => media.post)
-  medias?: Media[];
+  @Field(/* istanbul ignore next */ () => Media, { nullable: true })
+  @OneToOne(() => Media, (media) => media.post)
+  @JoinColumn()
+  medias?: Media;
```

Comment:
💡 You changed it to be `one` image, so the name should change too to avoid confusion

```suggestion
  media?: Media;
```